src/dynapi/SDL_dynapi_procs.h

```diff
@@ -509,6 +509,7 @@ SDL_DYNAPI_PROC(void,SDL_DetachThread,(SDL_Thread *a),(a),)
 SDL_DYNAPI_PROC(SDL_TLSID,SDL_TLSCreate,(void),(),return)
 SDL_DYNAPI_PROC(void*,SDL_TLSGet,(SDL_TLSID a),(a),return)
 SDL_DYNAPI_PROC(int,SDL_TLSSet,(SDL_TLSID a, const void *b, void (*c)(void*)),(a,b,c),return)
+SDL_DYNAPI_PROC(void,SDL_TLSCleanup,(void),(),)
```

Comment:
New api additions are not inserted, but appended to the existing list:
Run gendynapi.pl to achieve that goal.

Reply:
Xing Ji, can you please fix this

Reply:
OK